### PR TITLE
Added NOT LIKE support for WildcardFilters

### PR DESCRIPTION
**Pull request by mikeengland**
Sorry for the large diff on the test file due to hitting the reformat.

Also - changed the 'wildcard' filter method on the UniqueDimension to 'like' to make it more natural to use.

### CODE REVIEW

File: fireant/slicer/dimensions.py [hunk 1]
```diff
@@ -169,6 +169,21 @@ def wildcard(self, pattern):
...
+        :param pattern:
+            A pattern to match against the dimension's display definition.  This pattern is used in the SQL query as
+            the `NOT LIKE` expression.
+        :return:
+            A slicer query filter used to filter a slicer query to results where this dimension's display definition
+            matches the pattern.
+        """
+        if self.display_definition is None:
+            raise QueryException('No value set for display_definition.')
+        return NegatedWildcardFilter(self.display_definition, pattern)
```
Comment (twheys) [inline fireant/slicer/dimensions.py]: Maybe call this NotLikeFilter ?

File: fireant/tests/slicer/queries/test_builder.py [hunk 2]
```diff
@@ -258,10 +258,10 @@ def test_build_query_with_multiple_dimensions_and_visualizations(self):
         query = slicer.data \
             .widget(f.DataTablesJS([slicer.metrics.votes, slicer.metrics.wins])) \
             .widget(f.HighCharts(
-              axes=[
-                  f.HighCharts.PieChart([slicer.metrics.votes]),
-                  f.HighCharts.ColumnChart([slicer.metrics.wins]),
-              ])) \
+            axes=[
```
Comment (twheys) [inline fireant/tests/slicer/queries/test_builder.py]: please revert all of these changes to indentation

File: fireant/slicer/dimensions.py (hunk 1 above)
Comment (mikeengland) [reply fireant/slicer/dimensions.py]: Fixed

File: fireant/tests/slicer/queries/test_builder.py (hunk 2 above)
Comment (mikeengland) [reply fireant/tests/slicer/queries/test_builder.py]: Fixed

**Pull request merged**